### PR TITLE
chore: migrate Docker.ubuntu18/rhel8 file to build SCTPD with Bazel

### LINKED ISSUE #10891: bazel: Migrate OAI-Jenkins related Dockerfiles to build SCTPD/MME with Bazel (Dockerfile.rhel8)
We need to do something like https://github.com/magma/magma/pull/10886

### LINKED ISSUE #10892: bazel: Migrate OAI-Jenkins related Dockerfiles to build SCTPD/MME with Bazel (Dockerfile.ubuntu18.04)

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
Modify Jenkins Dockerfiles to use Bazel to build Sctpd. Thanks @rdefosse for helping me figure out what files to change. :) 

## Test Plan
CI

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/docker/mme/Dockerfile.ubuntu18.04 [hunk 1]
```diff
@@ -179,7 +185,7 @@ WORKDIR /
 RUN echo "Install fmtlib required by Folly" && \
     git clone https://github.com/fmtlib/fmt.git && cd fmt && \
     mkdir _build && cd _build && \
-    cmake .. && \
+    cmake .. -DFMT_TEST=0 && \
```
Comment (themarwhal) [inline lte/gateway/docker/mme/Dockerfile.ubuntu18.04:188]: speed up the build a bit by not building test directories

**Review by rdefosse — APPROVED** [review]

**Comment by themarwhal** [comment]
@rdefosse it looks like the job is still failing. Will take a look tomorrow! (It might just be checking for the SCTPD binary in the wrong place)

File: lte/gateway/docker/mme/Dockerfile.ubuntu18.04 [hunk 2]
```diff
@@ -8,6 +8,7 @@ ARG FEATURES=mme_oai
 ENV MAGMA_ROOT=/magma
```
Comment (arunuke) [inline lte/gateway/docker/mme/Dockerfile.ubuntu18.04:8]: Should we consider upgrading to Ubuntu 20.04?

**Review by arunuke — APPROVED** [review]
LGTM. Would be good to land on Ubuntu 20.04 (over 18.04).

File: lte/gateway/docker/mme/Dockerfile.ubuntu18.04 (hunk 2 above)
Comment (themarwhal) [reply lte/gateway/docker/mme/Dockerfile.ubuntu18.04:8]: cc @rdefosse

**Comment by themarwhal** [comment]
Blegh

**Pull request closed without merging**